docs/hello-neighbour.yaml

```diff
@@ -93,35 +93,33 @@ paths:
           description: "Church profile modified"
           schema:
             $ref: "#/definitions/ChurchIn"
-        404:
-          description: "Church not found"
-          schema:
-            $ref: "#/definitions/ErrorResponse"
         400:
           description: "Invalid input"
           schema:
-            $ref: "#/definitions/ErrorResponse"
+            $ref: "#/definitions/Error"
+        401:
+          $ref: "#/responses/Unauthenticated"
+        404:
+          description: "Church not found"
+          schema:
+            $ref: "#/definitions/Error"
     delete:
       tags:
       - "leader"
       summary: "Delete church"
-      description: "Delete a church from the system."
+      description: "Delete the leader's church from the system."
       operationId: "deleteChurch"
       produces:
       - "application/json"
-      parameters:
-      - in: "path"
-        name: "churchUuid"
-        type: "string"
-        format: "uuid"
-        required: true
       responses:
         204:
           description: "Church deleted"
+        401:
+          $ref: "#/responses/Unauthenticated"
         404:
           description: "Church not found"
           schema:
-            $ref: "#/definitions/ErrorResponse"
+            $ref: "#/definitions/Error"
```

Comment:
I'm not quite sure what this will do. Does it delete the church and all its users? Either way, I don't think there is a 404 for it because if there was no church, your account shouldn't actually work.

Reply:
Outdated response, but I assume deleting a church would disassociate the members (leaving their accounts intact)?

Reply:
I agree that that is probably the desired behaviour.